components/camel-ai/camel-langchain4j-chat/src/main/docs/langchain4j-chat-component.adoc

```diff
@@ -143,3 +143,37 @@ messages.add(new SystemMessage("You are asked to provide recommendations for a r
 
 String response = template.requestBody("direct:send-multiple", messages, String.class);
 ----
+
+== Chat with Tool
+Camel langchain4j-chat used as a consumer can be used to implement a Tool,
+right now Tools are supported only via the OpenAiChatModel
+backed by OpenAI APIs.
+
+Tool Input parameter can be defined via parameterName and parameterType in case of only one parameter,
```

Comment:
It would be good to use adoc formatting for better legibility when referring to parameters or options